tslint.json

```diff
@@ -1,20 +1,32 @@
 {
   "extends": "tslint:latest",
   "rules": {
-    "interface-name": false,
-    "max-classes-per-file": false,
-    "max-line-length": [true, 140],
-    "member-ordering": [true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
-    ],
-    "no-unused-variable": false,
-    "variable-name": [true,
-      "ban-keywords",
-      "check-format",
-      "allow-leading-underscore",
-      "allow-pascal-case"
-    ]
+    "interface-name": {
```

Comment:
It uses it from `node_modules`:
1. `package.json` refers it as `tslint` so this executable should be in path (which was done by npm) (just replace `lint` script to `which tslint`)

2. All rules in new format are just disabled (check `isEnabled` logic in `AbstractRule`)
Simply use 10 for `max-line-length` rule and there will be no violations. Restore previous format and you'll see all of them.

3. You can alter constructor in `node_modules/tslint/lib/language/rule/abstractRule.js` to print all not enabled rules - you'll see all of rules that has new format.